javalee/database/howToDo.md

```diff
@@ -0,0 +1,10 @@
+<h1>Step-by-step to use PostgreSQL</h1>
+
+<p>1º - Vá para https://www.postgresql.org;</p>
+<p>2º - Encontre o botão que leva para a tela de download;</p>
+<p>3º - Selecione seu sistema operacional (windows, linux, ios, etc);</p>
+<p>4º - Click no texto com link (cor azul) "Download the installer";</p>
+<p>5º - Execute o instalador e instale o programa;</p>
+<p>6º - Leia e aperte "next" de acordo com suas preferências;</p>
+<p>7º - Configure suas credentiais e mantenha a porta do SGBD como 5432;</p>
```

Comment:
Esse é um passo importante, você tem que definir as credenciais que todos devem usar. 
Qual user, qual password, qual endereço. Pq assim todos ficam com a uniformidade do banco.

Reply:
Mesmo se for aberto para todos devo por as credenciais no MD?